src/controllers/executionPlanWebviewController.ts

```diff
@@ -141,6 +146,10 @@ export class ExecutionPlanWebviewController extends ReactWebviewPanelController<
     }
 
     private async createExecutionPlanGraphs(content: string) {
+        sendActionEvent(
+            TelemetryViews.QueryPlan,
+            TelemetryActions.OpenQueryPlan,
+        );
```

Comment:
sending an event at the beginning can be useful, but we generally care more about the end state: did it succeed or fail?  if it failed, what error did we get?  how many execution plans were in the XML?  how long did generating the plan from the XML take?